lib/scripts.js

```diff
@@ -0,0 +1,18 @@
+let game;
+
+function startNewGame () {
+  game = new Game()
+}
+
+// invoke startNewGame on 'Start' button
+startNewGame()
+
+$('.catergory-text').text(game.getCategory());
+
+// $('#startGame').onClick(startNewGame)
+// $('#restartGame').onClick(startNewGame)
```

Comment:
Glad there is some pseudo-code,  so we can know what our next issues are for our code-base.